config-model/src/main/java/com/yahoo/vespa/model/ConfigProxy.java

```diff
@@ -47,6 +47,13 @@ public ConfigProxy(Host host) {
      */
     public int getPortCount() { return 1; }
 
+    @Override
+    public String[] getPortSuffixes() {
+        String[] suffixes = new String[1];
```

Comment:
Can be simplified to a single statement, `return new String[]{"rpc"}`. Applies to other methods that do this as well.